src/persistedState/getPersistedState.ts

```diff
@@ -28,7 +32,10 @@ export const getPersistedState = async (
 		const str = buffer.toString();
 		const json = JSON.parse(str);
 
-		const persistedStateEither = persistedStateCodec.decode(json);
+		const persistedStateEither = persistedStateCodec.decode({
```

Comment:
oh I get what you wanted to do now :)

https://github.com/Effect-TS/schema#default-values we can use this instead (and temporarily, until people migrate to the new version of the persisted state)